client/src/pages/index.js

```diff
@@ -1,5 +1,6 @@
 import Home from "./Home";
 import Room from "./Room";
 import Join from "./Join";
+import Profile from "./Profile";
 
-export { Home, Room, Join };
+export { Home, Room, Join, Profile };
```

Comment:
How do you feel about this index.js file? Is it useful?

Reply:
I think it's a reasonable alternative to having each imported component being its own line. We should keep it in for now.